pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = [
+    "setuptools",
+    "wheel",
+    "Cython>=0.28.5",
```

Comment:
Could you please update me why we need Cython (and maybe add it as a comment)? I thought the pyrfr uses SWIG and does therefore not require Cython.